torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -22,6 +22,38 @@ namespace rpc {
 
 namespace {
 
+py::object toPyObjInternal(RpcCommandBase& rpc, MessageType messageType) {
```

Comment:
No change to these two functions, I just moved them from `torch::distributed::rpc` to an anonymous namespace, as `rpc/init.cpp` no longer needs to access this function.

Reply:
Let create a followup PR on top to merge these two functions.

Reply:
#37536